package.json

```diff
@@ -7,15 +7,16 @@
     "name": "Cow Protocol"
   },
   "scripts": {
-    "build": "rm -rf dist && rm -rf types && yarn typechain && tsc",
+    "build": "rm -rf dist && rm -rf src/types/generated && yarn typechain && tsc",
     "fmt": "prettier --write \"src/**/*.ts\"",
     "lint": "eslint \"src/**/*.ts\" && prettier --check \"src/**/*.ts\"",
     "lint:fix": "eslint --fix \"src/**/*.ts\" && prettier --write \"src/**/*.ts\"",
     "test": "yarn build && jest ./dist",
-    "typechain": "typechain --target ethers-v5 --out-dir src/types/ \"abi/*.json\"",
+    "typechain": "typechain --target ethers-v5 --out-dir src/types/generated/ \"abi/*.json\"",
     "prepare": "husky install && yarn typechain"
```

Comment:
can you add a script to run the cli easily? 

Also, we should probably add a bin section https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bin

Reply:
Tracking in #60